srcs/requirements/frontend/src/navigation.ts

```
@@ -0,0 +1,172 @@
+            addRankingEvents();
+            break;
+    }
+}
+
+/**
+ * @brief Adds event listeners for the login view.
+ * 
+ * This function sets up the event listener for the login button, which navigates to the home view upon click.
+ */
+function addLoginEvents(): void {
+    document.getElementById("login-button")!.addEventListener("click", () => {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)